packages/webapp/pages/upvoted.tsx

```diff
@@ -5,12 +5,14 @@ import {
   getMainFeedLayout,
   mainFeedLayoutProps,
 } from '../components/layouts/MainFeedPage';
-import { defaultOpenGraph, defaultSeo } from '../next-seo';
+import { defaultOpenGraph } from '../next-seo';
+import { getTemplatedTitle } from '../components/layouts/utils';
 
 const seo: NextSeoProps = {
-  title: 'Most upvoted posts on daily.dev',
+  title: getTemplatedTitle('Most upvoted posts for developers'),
```

Comment:
This change is not in the specs doc, so waiting to confirm here: https://dailydotdev.slack.com/archives/C06JCC41KSS/p1728371071348349?thread_ts=1728212815.310049&cid=C06JCC41KSS